src/app/src/reducers.js

```diff
@@ -1,18 +1,24 @@
 import { createReducer } from 'redux-act';
 import update from 'immutability-helper';
 
-import { hideScreensaver, showScreensaver } from './actions';
+import { changeAboutSlide, hideScreensaver, showScreensaver } from './actions';
 
 export const initialState = {
+    aboutSlideIndex: 0,
     isScreensaverVisible: true,
 };
 
 export default createReducer(
     {
+        [changeAboutSlide]: (state, payload) =>
```

Comment:
Nice job adding this action & reducer set! I think we can tweak the name to give its function some clarity. What do you think about "setAboutSlideIndex" or "saveAboutSlideIndex"?